lib/engines/with_migrations.rb

```diff
@@ -0,0 +1,24 @@
+module Engines
+  # It DRYies the common snippet to append engine's migrations to the main app
+  module WithMigrations
```

Comment:
It turns out this is the way to go with migrations in CBRA :point_right:  http://blog.pivotal.io/pivotal-labs/labs/leave-your-migrations-in-your-rails-engines.

I think I'll extract it as a gem and add it as a dependency. What do you think? I'll probably do the same in Redbooth, btw.